web/cobrands/sass/_base.scss

```diff
@@ -1493,6 +1493,27 @@ html.js #map .noscript {
       background-image:url($image-sprite);
       background-position: flip(right, -341px) -4070px;
     }
+    &#fms_shortlist_all {
```

Comment:
Not necessarily now (though could be), but as we have factored out staff JS, should we also do the same with staff CSS?